crates/theme/src/registry.rs

```diff
@@ -124,24 +175,28 @@ impl ThemeRegistry {
     }
 
     pub fn load_user_themes(&mut self) {
-        #[cfg(not(feature = "importing-themes"))]
-        self.insert_user_theme_families(crate::all_user_themes());
+        let theme_paths = self
+            .assets
+            .list("themes/")
+            .unwrap()
+            .into_iter()
+            .filter(|path| path.ends_with(".json"));
+
+        for path in theme_paths {
+            let theme = self
+                .assets
+                .load(&path)
+                .expect(&format!("Failed to load theme '{path}'"));
+
+            let theme_family: ThemeFamilyContent = serde_json::from_slice(&theme).unwrap();
+
+            self.insert_user_theme_families([theme_family]);
+        }
     }
 }
 
 impl Default for ThemeRegistry {
     fn default() -> Self {
-        let mut registry = Self {
-            themes: HashMap::default(),
-        };
-
-        // We're loading our new versions of the One themes by default, as
-        // we need them to be loaded for tests.
-        //
-        // These themes will get overwritten when `load_user_themes` is called
-        // when Zed starts, so the One variants used will be the ones ported from Zed1.
-        registry.insert_theme_families([crate::one_themes::one_family()]);
-
-        registry
+        Self::new(Box::new(()))
```

Comment:
I'm not sure if we still need this `Default` impl, but I erred on the side of caution and left it in place, using the `AssetSource` impl on `()` to load the empty asset source (which should never get called).